dali/test/python/test_operator_coin_flip.py

```diff
@@ -54,12 +48,10 @@ def check_coin_flip(device='cpu', batch_size=32, max_shape=[1e5], p=None, use_sh
         pipe.set_outputs(*outputs)
     pipe.build()
     outputs = pipe.run()
-    data_out = outputs[0].as_cpu() \
-        if isinstance(outputs[0], TensorListGPU) else outputs[0]
+    data_out = outputs[0].as_cpu() if isinstance(outputs[0], TensorListGPU) else outputs[0]
```

Comment:
The breaks were intentionally introduced for readability :\

Reply:
It is pretty plain and obvious Python's ternary, I'd prefer to keep it this way. @awolant , what's your opinion here?